operation.go

```diff
@@ -69,6 +69,13 @@ func (operation *Operation) ParseComment(comment string, astFile *ast.File) erro
 	switch lowerAttribute {
 	case "@description":
 		operation.ParseDescriptionComment(lineRemainder)
+	case "@tag.description.markdown":
```

Comment:
I think you misunderstood the feature #506. Your code has nothing to do with it.

Also, you need to add "usage" details it to readme.md

Reply:
yes, was wrong...please check the my other comment...
https://github.com/swaggo/swag/pull/539#issuecomment-540781023